README.md

```diff
@@ -1,4 +1,41 @@
 # Express server for TasteRX
 
-This API is meant to be paired with a React frontend hosted here:
-https://github.com/dstavis/tasterx
\ No newline at end of file
+[Deployed UI](https://tasterx.herokuapp.com/)
+[Deployed API](https://tasterx-api.herokuapp.com/)
+
+This API is meant to be paired with the React UI hosted here:
+https://github.com/dstavis/tasterx
+
+## Abstract
+
+TasteRX is an engaging UX born from the idea of curing the world of bad taste.  Do you have good taste in TV shows?  Do you think the world would be a better place if more people watched a particular show?  The TasteRX UI allows users to search for and then 'prescribe' a TV show with a custom message to a fellow human; the created prescription can then easily be shared via URL.   
+
+This project utilized asynchronous workflow relying on code reviews within PR requests, daily stand-ups, project board tickets, and Slack among four remote contributors
+
+* Learning Goals:
+    * Explore backend web technologies to develop a functioning database that would allow our application to be accessed on any device with an internet connection.   
+    * Gain further familiarity with React, React Router, and E2E testing with Cypress
+    * App development using React hooks in place of class components
+
+## Technologies Used
+- Postgres
+- Heroku
+- Sequelize
+- Express.js 
```

Comment:
I think this may just be Express. The url says `expressjs`, but the website says Express for Node.js